scripts/docker-entrypoint.sh

```diff
@@ -88,6 +88,14 @@ if [ "$1" = 'mgmtapi' ]; then
         echo "JVM_OPTS=\"\$JVM_OPTS -javaagent:${MGMT_AGENT_JAR}\"" >> ${CASSANDRA_CONF}/cassandra-env.sh
     fi
 
+    # Add CDC agent
+    CDC_AGENT_JAR="$(find "${CDC_AGENT_PATH}" -name cdc-agent.jar)"
```

Comment:
This is the line that gets way more complicated if we have version specific things in the file name. But, if it's important to preserve that (like c3/c4 or pulsar/luna or whatever), we can change this to make it work.